File: Python/calculadora.py
import sys

print("Menu")
print("1- Suma")
print("2- Resta")
print("3- Multiplicacion")
print("4- Division")
opcion = int(input("Elige una opcion..."))

if (opcion > 0 and opcion < 5):
    num1 = int(input("Ingresa el primer numero..."))
    num2 = int(input("Ingresa el segundo numero..."))
    if (opcion == 1):
        resultado = num1 + num2
    elif (opcion == 2):
	resultado = num1 - num2
    elif (opcion == 3):
	resultado = num1 * num2
    elif (opcion == 4):
	resultado = num1 / num2

    print"El resultado de la operacion es: ", resultado
else:
    print("Opcion invalida")

File: C/calculadora.c
#include <stdio.h>

int main() {

  int num1,num2,resultado;
  int opcion;

  printf("Menu\n\n");
  printf("1- Suma\n");
  printf("2- Resta\n");
  printf("3- Multiplicacion\n");
  printf("4- Division\n");
  printf("Elige una opcion...: ");

  scanf("%i",&opcion);

  if (opcion > 0 && opcion < 5) {
    printf("Ingresa el primer elemento: ");
    scanf("%i",&num1);
    printf("Ingresa el segundo elemento: ");
    scanf("%i",&num2);

  
      if (opcion == 1) {
        resultado = num1 + num2;
      } else if (opcion == 2) {
        resultado = num1 - num2;
      } else if (opcion == 3) {
        resultado = num1 * num2;
      } else if (opcion == 4) {
        resultado = num1 / num2;
      }

      printf("\nEl resultado de la operacion es: %i\n", resultado);
  } else {
    printf("Opcion invalida\n"); 
  }

    
  return 0;

}




























 
